samples/TestWare.Samples.Selenium.Web/POM/Login/LoginPage.cs

```diff
@@ -47,7 +47,7 @@ public void CheckUserIsAtHomepage()
         RetryPolicies.ExecuteActionWithRetries(
             () =>
             {
-                this.Driver.Url.Should().Be("https://demo.guru99.com/V4/manager/Managerhomepage.php");
+                this.Driver.Url.Should().Be("https://www.saucedemo.com/inventory.html");
```

Comment:
This method is not CheckUserIsAtHompage.
This method should be moved to an Inventory page.

Reply:
Done